tools/roslaunch/src/roslaunch/resource_cache.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import roslib.packages
+import catkin.find_in_workspaces
+
+from catkin.workspace import get_source_paths, get_workspaces
+from catkin_pkg.packages import find_packages
+
+
+def _get_source_path_to_packages(workspaces=None,
+                                 workspace_to_source_spaces=None):
+    """Returns dictionay with mappings from source paths to packages.
```

Comment:
Sorry for the late tiny feedback but some of it only appears when I open the code in my editor which does all kind of style checks.

Please change first word from `Returns` to `Return` (PEP257 D401).
Please add an empty line between this and the next line (PEP257 D205).